.travis.yml

```diff
@@ -7,12 +7,12 @@ python:
 
 env:
   matrix:
-  - ANSIBLE_REF=2.4.3.0
-  - ANSIBLE_REF=2.3.3.0
+  - ANSIBLE_REF=2.6
+  - ANSIBLE_REF=2.5
```

Comment:
it is not matching the name of your PR, is it OK ? :)

Reply:
could you change this to `ANSIBLE_REF="ansible<2.6"` here ?
it seems easier to read/understand (to me)

Reply:
Indeed ! Updated:
- 2.5.0: https://travis-ci.org/peopledoc/ansible-boot-lxc/jobs/359266622#L740
- 2.4.3.0: https://travis-ci.org/peopledoc/ansible-boot-lxc/jobs/359266623#L744